src/components/ceremony/handlers/create-internship-engagement-default-ceremony.handler.ts

```diff
@@ -0,0 +1,21 @@
+import { EventsHandler, IEventHandler } from '../../../core';
+import { InternshipCreatedEvent } from '../../engagement/events';
+import { CeremonyService } from '../ceremony.service';
+
+@EventsHandler(InternshipCreatedEvent)
+export class CreateInternshipEngagementDefaultCeremonyHandler
+  implements IEventHandler<InternshipCreatedEvent> {
+  constructor(private readonly ceremonies: CeremonyService) {}
+
+  async handle({
+    internshipEngagement,
+    input,
+    session,
+  }: InternshipCreatedEvent) {
+    await this.ceremonies.attachInternshipEngagement(
```

Comment:
There's no point in cluttering the CeremonyService more. The new logic can and should go in this file.
(It's ok to still call `ceremonies.create()`)

Reply:
I refactored this.